### ISSUE TITLE
Some issue

### ISSUE THREAD

**Issue by Shiwo** [chummer5a/chummer5a#247]
Greetings ! 

Glad to see Chummer 5 still updated, respect for it ! 

I would like to report some issue we found with Build 167. Sorry if already reported / solved.

- During character creation (priority system), issue with Skill ranks increased with Karma (Buildpoint). When I save the character still in build mode, a BP automatically added as well to them. So when I open the saved character, the rank will be 2 instead of 1 (and and will have less BP, even going into below 0).
Same issue with the Knowledge Skills.

- Contact Points
In the Build Summary on the right sometimes it is not updating, only after restart.
Right from the "Add Contact" button the update is working, but sometimes it shows incorrect value.
It shows the correct Charisma x3 value or simply Charisma value or Charisma x 2 value, changing each time, when I load the character.
Well, using the House rule "Free Contact Point equals to Charisma" can "solve" this issue.

- Exceed Positive / Negative Qualities Karma
It say "Allow characters to exceed 35 worth of Positive/Negative Qualities"
Well.. only 25 is allowed as default, not 35. And even check them you are still not able to exceed the 25.

- Advanced Lifestyle has strange calculation
Created a Lifestyle: 0 Roommates, 100% pay, calculating 7072 nujen. -> Push OK, seems OK, 7072 nujen / month
Same Lifestyle : 3 Roommates, 100% pay, calculating 8656 nujen. -> Push OK, will be 11253 nujen / month
Same Lifestyle : 3 Roommates, 25% pay, calculating 2168 nujen. -> Push OK, will be 705 nujen / month
Same Lifestyle : 0 Roommates, 25% pay, calculating 1762 nujen. -> Push OK, will be 440 nujen / month
An another issue with the Advanced Lifestyle, Edit set it back to default, like adding a brand new one.

- Linguist Quality 
Based on the description, it should be work like the College Education (RF) or School of Hard Knocks Quality.
These 2 seems fine.I can buy 2 ranks for 1 BP of Academic or Street Knowledge. But the Linguist not works, I still have to pay 2 BP for Rank 2 Language for example.

+1 Proposal
Please add a "Show Qualities except Metagenetic" to the Quality Selection windows.

Regards,
Shiwo

**Comment by joha4270** [comment]
Most of this is fixed or have an open issue

Qualities 35: I was aware of this issue even if no open issue report existed. Still, might actually bother to fix it now :)

Linguist: Something new 

"Show Qualities except Metagenetic"
Should be easy enough to do

**Comment by chummer5a** [comment]
Linguist issue is fixed in 5.168. 

Lifestyles need to be gutted and redone, there's some calculations for the standard lifestyle overwriting the calculations for the advanced lifestyles. Should be fixed over the weekend.

English Language file for Exceeding Qualities has been set to 25. Fixed in 5.168.

Skill rank issue is also fixed in 5.168.

**Comment by Shiwo** [comment]
"English Language file for Exceeding Qualities has been set to 25. Fixed in 5.168."

- Still 35 with 5.168
I have two English (US) language right now, tried both, even changed to German and back to English, but remained 35.
And still not allow me to exceed the limit, even I checked these options.


- Additional issue : 
During Karma mod creation it is not possible to set a Native Language.
1 Karma is the minimum and the Language will have a rank in any way.

**Comment by joha4270** [comment]
Still 35 with 5.168
I have two English (US) language right now, tried both, even changed to German and back to English, but remained 35.
And still not allow me to exceed the limit, even I checked these options.

This is because the update function fails to update the language file. If you reinstall chummer from scratch this will be fixed

Please create a new issue for the last one.
The idea is one issue is one problem/feature not a thread discussing 7 different things :P
